templates/components/carousel.html

```diff
@@ -1,16 +1,22 @@
 <section class="heroCarousel"
     data-slick='{
-        "arrows": {{arrows}},
         "mobileFirst": true,
         "slidesToShow": 1,
         "slidesToScroll": 1,
         "autoplay": true,
         "autoplaySpeed": {{carousel.swap_frequency}},
         "lazyLoad": "anticipated",
-        "accessibility": false
+        "slide": ".hero-slide",
```

Comment:
I suggest to name it as js-'name' as well

Reply:
agree